src/FixedDataTableCell.js

```diff
@@ -253,6 +254,7 @@ class FixedDataTableCell extends React.Component {
           className={cx('fixedDataTableCellLayout/columnResizerContainer')}
           style={columnResizerStyle}
           onMouseDown={this._onColumnResizerMouseDown}
+          onDoubleClick={this._onColumnResizerDoubleClick}
```

Comment:
> How does `onMouseDown` work when the user double clicks the resizer? Is it going to be called twice? Do you see any issues that might pop up (eg: does the resize event trigger twice?)
Yes the resize does gets called twice, right now I have handled it to ignore the event if the new column width and current column width is same. Can we do something similar in fdt only. I also think this can potentially leave the column resize blue bar to not re render if we do not trigger mouse up?

The simplest way will be to leave upto the consumer to decide here similar to the way I am handling in my use case.

Reply:
Probably in this [line](https://github.com/schrodinger/fixed-data-table-2/blob/master/src/ColumnResizerLine.js#L170), we can just do setState and do not call onResizeEnd if the props initialWidth and state width are equal?

Reply:
Okay, I see there is a need to re render FDT with prop isColumnResizing set as false  because of this [line](https://github.com/schrodinger/fixed-data-table-2/blob/ad60599a46015280452c85851a7c20dac3cf606a/src/reducers/columnStateHelper.js#L50)

Reply:
Okay after taking a deeper look, I think we should introduce a COLUMN_RESIZE_END action type similar to COLUMN_RESIZE and call it on when column resize ends. This new action type can set isColumnResizing as false and also clear the columnResizing data in the state and at the same time call decide to whether to call the user passed onColumnResizeEnd callback. This approach has two benefits, it takes away need to re render fdt to remove the column reisizer blue line and also will not call onColumnResizerEnd callback unnecessarily when the new and current column width are equal. 

Let me know your thoughts on this

Reply:
> I think we should introduce a COLUMN_RESIZE_END action type similar to COLUMN_RESIZE and call it on when column resize ends. This new action type can set isColumnResizing as false and also clear the columnResizing data in the state and at the same time decide to whether to call the user passed onColumnResizeEnd callback.

This sounds good to me. We should probably leave it up to the user to decide if they want to ignore the `onColumnResizeEnd` callback when the new and old widths are identical.

> This new action type can set isColumnResizing as false and also clear the columnResizing data in the state

I got a question here: will there be any scenarios where the user doesn't want `isColumnResizing` to be false?
Maybe they want to ignore the current resize due to some reason, so in this case they'll plan to keep `isColumnResizing` as true. 
Ideally we want the user to be able to control column resizing through the `isColumnResizing` prop but I'm not aware of any examples in the wild that use the prop, nor do I know how exactly it is made use of in the wild...